docs/3-cloud-computing/3.3.5-aci.md

```diff
@@ -49,6 +49,8 @@ EOF
 
 3. Build the Docker image locally.
 
+?> ACI architecture is x86. Make sure your container is compiled for x86. `docker inspect --format='{{.Os}}/{{.Architecture}}' <my_image>` should show `linux/amd64`
```

Comment:
Liatrio issues M1/M2 macs and images compiled for arm were not working on ACI and giving strange errors

Reply:
Add in a note about the `--platform linux/amd64` flag also?

Reply:
Intentionally left this hint somewhat vague. As I want them to look into multi-architecture docker builds